README.md

```diff
@@ -54,73 +54,29 @@ Optional fields: `links` and `milestones`
 
 ```json
 {
-  "name": "Eddie Jaoude",
+  "name": "YOUR NAME",
   "type": "personal",
-  "bio": "Founder of EddieHub",
-  "avatar": "https://github.com/eddiejaoude.png",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
   "links": [
     {
       "name": "Follow me on GitHub",
-      "url": "https://github.com/eddiejaoude",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
       "icon": "github"
     },
     {
       "name": "Follow me on Twitter",
-      "url": "https://twitter.com/eddiejaoude",
+      "url": "https://twitter.com/YOUR_TWITTER_USERNAME",
       "icon": "twitter"
-    },
-    {
-      "name": "Learn more about Open Source on my YouTube channel",
-      "url": "https://youtube.com/eddiejaoude",
-      "icon": "youtube"
-    },
-    {
-      "name": "Follow me on LinkedIn",
-      "url": "https://linkedin.com/in/eddiejaoude",
-      "icon": "linkedin"
```

Comment:
Many people have LinkedIn and YouTube as well. We should keep this and adjust it to be a good example, IMO.

Reply:
> We have seen ppl not havin yt/linkedin either using Eddie's data or reaching out to use asking what has to be added 😅

You can tell them that those fields are optional, something like:

```json
    {
      "name": "Learn more about Open Source on my YouTube channel", ## If you have a YouTube channel, use this with a description of yours and your link.
      "url": "https://youtube.com/eddiejaoude",
      "icon": "youtube"
    },
```

Not sure if that's the correct way of adding comments in JSON, but that should be good. 🙂

Reply:
You can't have comments in JSON 🙂

Reply:
Then we can mention that in the readme?